content-repo/extra-docs/packs/content-management.md

```diff
@@ -828,3 +828,53 @@ Currently, the pack does not support the following features:
     Yes, that is supported.
 
 
+
+## Migration Guide from Dev-Prod to CI/CD
+
+This guide will help you migrate from the Dev-Prod environment to the CI/CD environment. In the process, you will need to download all your custom content from the Prod environment and also to delete some types of content from the Prod environment, to avoid conflicts.
```

Comment:
```suggestion
This guide will help you migrate from the Dev-Prod environment to the CI/CD environment. In the process, you will need to download all your custom content from the Prod environment and also to delete some types of content from the Prod environment to avoid conflicts.
```